src/Particle/LongRange/tests/test_StructFact.cpp

```diff
@@ -48,22 +48,22 @@ TEST_CASE("StructFact", "[lrhandler]")
   ref.R[2] = {0.3, 4.0, 1.4};
   ref.R[3] = {3.2, 4.7, 0.7};
 
-  REQUIRE(simulation_cell.getKLists().numk == 263786);
+  REQUIRE(simulation_cell.getKLists().getNumK() == 263786);
   StructFact sk(ref.getLRBox(), simulation_cell.getKLists());
   sk.updateAllPart(ref);
 
   CHECK(sk.rhok_r.rows() == ref.groups());
   CHECK(sk.rhok_i.rows() == ref.groups());
-  CHECK(sk.rhok_r.cols() == simulation_cell.getKLists().numk);
-  CHECK(sk.rhok_i.cols() == simulation_cell.getKLists().numk);
+  CHECK(sk.rhok_r.cols() == simulation_cell.getKLists().getNumK());
+  CHECK(sk.rhok_i.cols() == simulation_cell.getKLists().getNumK());
 
   std::vector<std::complex<double>> rhok_sum_ref{-125.80618630936, 68.199075127271};
 
   for (int i = 0; i < ref.groups(); i++)
   {
-    std::complex<double> rhok_sum, rhok_even_sum;
```

Comment:
My fix https://github.com/QMCPACK/qmcpack/pull/5382/files needs to be reinstated .